src/api.ts

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+
+const API_BASE_URL = 'http://sdtest.wz.cz/api';
+
```

Comment:
Don't forget to change it for prod
This will also not work after build due http and axios